diff --git a/app/gui/qt/CMakeLists.txt b/app/gui/qt/CMakeLists.txt
index b453b74eb..530e329ad 100644
--- a/app/gui/qt/CMakeLists.txt
+++ b/app/gui/qt/CMakeLists.txt
@@ -123,9 +123,9 @@ target_include_directories(${APP_NAME}
 
 target_link_libraries(${APP_NAME}
     PRIVATE
-    QScintilla
-    Qt5::Core 
-    Qt5::Gui 
+    qscintilla2_qt5
+    Qt5::Core
+    Qt5::Gui
     Qt5::Widgets
     Qt5::OpenGL
     Qt5::Concurrent
@@ -154,7 +154,7 @@ if(WIN32)
     add_custom_command(TARGET ${APP_NAME} POST_BUILD
         COMMAND ${CMAKE_COMMAND} -E make_directory ${SONIC_PI_ROOT}/app/server/native/osmid
         COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/external/osmid-prefix/src/osmid-build/Release/m2o.exe ${SONIC_PI_ROOT}/app/server/native/osmid
-        COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/external/osmid-prefix/src/osmid-build/Release/o2m.exe ${SONIC_PI_ROOT}/app/server/native/osmid)  
+        COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/external/osmid-prefix/src/osmid-build/Release/o2m.exe ${SONIC_PI_ROOT}/app/server/native/osmid)
 endif() # Win32
 
 # Make convenient source groups in the IDE
@@ -163,5 +163,5 @@ source_group(Osc FILES ${OSC_SOURCES})
 source_group(Scintilla FILES ${EDITOR_SOURCES})
 source_group(Qt FILES ${QT_SOURCES})
 
-source_group(Qt\\AutoMoc FILES ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}_automoc.cpp ) 
-source_group(Qt\\AutoMoc REGULAR_EXPRESSION "(mocs_*|qrc_.*|QtAwesome.*)" ) 
+source_group(Qt\\AutoMoc FILES ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}_automoc.cpp )
+source_group(Qt\\AutoMoc REGULAR_EXPRESSION "(mocs_*|qrc_.*|QtAwesome.*)" )
diff --git a/app/gui/qt/external/CMakeLists.txt b/app/gui/qt/external/CMakeLists.txt
index 9b4f2544d..e719d8d8e 100644
--- a/app/gui/qt/external/CMakeLists.txt
+++ b/app/gui/qt/external/CMakeLists.txt
@@ -3,9 +3,9 @@ include(ExternalProject)
 ExternalProject_Add(osmid
     PREFIX osmid-prefix
     SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/osmid
-    INSTALL_COMMAND "" 
+    INSTALL_COMMAND ""
     BUILD_COMMAND ${CMAKE_COMMAND} --build . --config Release
     )
 
-add_subdirectory(QScintilla-2.11.4)
+# add_subdirectory(QScintilla-2.11.4)
 
diff --git a/app/gui/qt/mainwindow.cpp b/app/gui/qt/mainwindow.cpp
index 2c02d1713..0d1e20d81 100644
--- a/app/gui/qt/mainwindow.cpp
+++ b/app/gui/qt/mainwindow.cpp
@@ -1038,7 +1038,7 @@ QString MainWindow::rootPath() {
     return QCoreApplication::applicationDirPath() + "/../../../../..";
 #else
     // On linux, CMake builds app into the build folder
-    return QCoreApplication::applicationDirPath() + "/../../../..";
+    return QCoreApplication::applicationDirPath() + "/../../..";
 #endif
 }
 
@@ -1658,7 +1658,7 @@ void MainWindow::scope() {
     scopeAct->setIcon( theme->getScopeIcon(piSettings->show_scopes));
     if(piSettings->show_scopes)
     {
-        for( auto name : scopeInterface->GetScopeCategories()) 
+        for( auto name : scopeInterface->GetScopeCategories())
         {
             scopeInterface->EnableScope( name, piSettings->isScopeActive(name));
         }
